docs/Performance/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Performance
+permalink: /performance/
+has_children: true
+nav_order: 6
+---
+
+## Performance
+
+This chapter focuses on asynchronous backend OData calls. It shows how you can make your OData calls "Promise ready".
+
+### Normal ECMAScript 5 Projects
+
+> IE11 compatible
+
+Find out how to wrap your OData calls with [Promises](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises) and make use of the Promise Chain feature.
+
+### Modern ECMAScript 6 Projects
+
+> Requires modern Browsers
+
+If your project uses modern Javascript, you can consume Promises using the [Async / Await](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await) pattern and handle OData calls as if they where synchronous.
```

Comment:
JavaScript? But not sure whether this would get too awkward to follow through with...

Reply:
Changed with the latest commit